### ISSUE TITLE
Add "Current transaction date" to explorer

### ISSUE THREAD

**Issue by Sajjon** [radixdlt/mtps#7]
It would be nice to see the date of the transactions currently being processed. Kind of goes along with the progress percentage measurement. As in "Currently processing transactions from 2014-05"

**Comment by MuncleUscles** [comment]
It won't be very accurate, as the different nodes progress through time at wildly different rates

**Comment by blenessy** [comment]
Change "Watch us reconstruct your transaction history at hyperspeed" to 
"Search for your Bitcoin transaction history up to:

DD-MMM-YYYY"

DD-MMM-YYYY is getting updated in real-time.

**Comment by laszlourszuly** [comment]
This change costs much more than what it's worth in my opinion.

We need to request the "latest-processed-bitcoin-transaction" atom from a node (which one? See @MuncleUscles's comment above) and with the current API we need to know the address of that atom (see `/api/atoms?...` API implementation in core - `address` is required) which we don't know at this point, as we don't know which atom it is yet.

Basically: "gimme latest atom" isn't specific enough for the node to know what to return.

**Comment by MuncleUscles** [comment]
I mean, you could just estimate very roughly based on the progress and https://www.blockchain.com/charts/n-transactions-total?timespan=all
